hera_mc/cm_hookup.py

```diff
@@ -84,9 +86,10 @@ def get_hookup(self, hpn, rev, port, at_date, state_args, exact_match=False):
         else:
             hookup_dict = self.__add_hookup_timing(hookup_dict)
             hookup_dict['columns'] = self.__get_column_headers(hookup_dict['hookup'])
-            if state_args['show_levels']:
-                hookup_dict = self.__hookup_add_correlator_levels(
-                    hookup_dict, state_args['levels_testing'])
+            if 'show_levels' in kwargs.keys() and kwargs['show_levels'] is True:
```

Comment:
This chunk of the code still references the `kwargs` variable, so it will crash if this code path is triggered.